### PR TITLE
People management data layer

### PR DESCRIPTION
**Pull request by oguzkocer**
This PR adds the data layer for the people management feature. There might be more things that we'll need, but I am making an effort to keep the PRs small from the start and since this is not getting into `develop` I think it's fine if something else comes up. Once everything is merged into `feature/people-management-sync` branch we can fully review the feature.

Needs review: @hypest

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                + "person_id               INTEGER PRIMARY KEY DEFAULT 0,"
+                + "user_name               TEXT,"
+                + "first_name              TEXT,"
+                + "last_name               TEXT,"
+                + "display_name            TEXT,"
+                + "avatar_url              TEXT,"
+                + "role                    TEXT)");
+    }
+
+    private static void dropTables(SQLiteDatabase db) {
```
Comment (hypest) [inline WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java]: Hey @oguzkocer , should I assume that this method will be used in some subsequent commits/PR? If yes, just ignore this comment. If not, can this be an indication of a `reset()` method missing maybe?

Comment (oguzkocer) [inline WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java]: Good catch @hypest! I totally forgot to add the `reset()` method. I am still not sure if we need these 2 methods before-hand though. I've seen other classes have this and I added it to keep it consistent. Maybe the ones I've checked did need them, I am not sure.

**Comment by oguzkocer** [comment]
Closing this PR to figure out the networking layer. Once I do that and satisfied with the data layer, I'll reopen this PR.

**Comment by oguzkocer** [comment]
Just adding a note here: Please DO NOT delete the feature branch, I'll open the PR again at a later time.

**Pull request closed without merging**